deploy_win.py

```diff
@@ -2,6 +2,8 @@
 
 import sys
 
+sys.path.append(".\..\..\src")
```

Comment:
This line seems strange. Can it be remediated? Is it still necessary? If so, can you add a comment explaining exactly what it is for?

Reply:
STLib, the submodule of ST, tries importing ST.That would a "poetry add ../../" which isnt what we want

this is a hack to include it anyway.

(ill add this as a comment)